File: PRAK503_2310817210022_MUHAMMAD_ADH-DHIYA_US_SALIM.py
# Fungsi untuk menentukan nilai maksimal
def maksimal(a, b):
    return a if a > b else b

# Fungsi untuk menentukan nilai minimal
def minimal(a, b):
    return a if a < b else b

bilangan = int(input())
nilai_list = list(map(int, input().split()))

maks = -100000
minim = 100000

for nilai in nilai_list:
    maks = maksimal(maks, nilai)
    minim = minimal(minim, nilai)

print(f"{maks} {minim}")

File: PRAK503_2310817210022_MUHAMMAD_ADH-DHIYA_US_SALIM.c
#include <stdio.h>

// Fungsi untuk menentukan nilai maksimal
int maksimal(int a, int b) {
    return (a > b) ? a : b;
}

// Fungsi untuk menentukan nilai minimal
int minimal(int a, int b) {
    return (a < b) ? a : b;
}

int main() {
    int bilangan, nilai;
    int maks = -100000;
    int minim = 100000;

    scanf("%d", &bilangan);

    for (int batas = 0; batas < bilangan; batas++) {
        scanf("%d", &nilai);
        maks = maksimal(maks, nilai);
        minim = minimal(minim, nilai);
    }

    printf("%d %d", maks, minim);

    return 0;
}
